packages/alert-banner/style.module.css

```diff
@@ -4,6 +4,9 @@
  */
 
 .root {
+  --token-color-foreground-primary: #3b3d45;
+  --token-color-foreground-faint: #656a76;
```

Comment:
Are we declaring these variables because we don't have access to the tokens in `react-components`? 

Either way, I wonder if we should remove this, reference the tokens in the declarations below with a fallback to ☝️ hex?

```
color: var(--token-color-foreground-faint, #656A76);
```
This would remove duplication, ensure a token is being used and then, in cases like `react-components` where we don't have the tokens, display the fallback. Thoughts?

Reply:
I like this approach much better, thanks for the idea!